src/renderer/screens/Preferences/components/PreferenceSwitch.js

```diff
@@ -20,12 +20,15 @@ import React from "react";
 import FormControlLabel from "@mui/material/FormControlLabel";
 import ListItem from "@mui/material/ListItem";
 import ListItemText from "@mui/material/ListItemText";
+import Skeleton from "@mui/material/Skeleton";
 import Switch from "@mui/material/Switch";
 import { useTranslation } from "react-i18next";
 
 const PreferenceSwitch = (props) => {
   const { t } = useTranslation();
 
+  const loaded = props.loaded === undefined ? true : props.loaded;
```

Comment:
Is this logic really "if props.loaded is undefined then loaded is true, otherwise go with the value of props.loaded? 

I know we've used this pattern somewhere else before, but as written, it doesn't seem to make any sense.

in isolation, "if props.loaded is undef, then loaded is true" just sounds like it's a bug, even if it's really what we want.

 It probably wants a comment or something. and some parens would probably help the human parsing.